recipe/meta.yaml

```diff
@@ -13,23 +13,24 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
+    - {{ compiler('c') }}
+  host:
     - python
-    - toolchain
+    - pip
+    - numpy 1.9.*     # [not (win and py36)]
+    - numpy 1.11.*    # [win and py36]
```

Comment:
Not sure how important `numpy` version `1.9` support is for us here. If not at all important, we can change this to just `numpy 1.11.*` (without selectors) or similar.